<launch>
    <node name="planar_segmentation" pkg="filters" type="planar_segmentation" output="screen">
        <param name="subscribed_topic" type="string" value="/cloud_cleaned"/>
        <param name="published_topic" type="string" value="planar_segmentation_result"/>
        <param name="optimizeCoefficients" type="bool" value="true"/>
        <param name="distanceThreshold" type="double" value="0.01"/>
    </node>
</launch>
